src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
 
 import { transparentize } from 'polished';
+import { useNavigate } from 'react-router';
 import { useParams } from 'react-router-dom';
 
-import { makeStyles, Avatar, Button } from '@material-ui/core';
+import { makeStyles, Button } from '@material-ui/core';
 
+import { Avatar } from '../../ui/Avatar/Avatar';
```

Comment:
can you add `export * from './Avatar/Avatar'` to `src/ui/index`? then this import can become
```
import { Avatar } from 'ui';
```